runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ScreenMarkerPlugin.java

```diff
@@ -216,6 +216,11 @@ public void finishCreation(boolean aborted)
 		{
 			final ScreenMarkerOverlay screenMarkerOverlay = new ScreenMarkerOverlay(marker);
 			screenMarkerOverlay.setPreferredLocation(overlay.getBounds().getLocation());
+
+			// Ensure created overlay is of at least minimum width/height
+			final Rectangle markerBounds = overlay.getBounds();
```

Comment:
This should just be checked in Overlay class (for example override setPrefferedSize there and check it also there)

Reply:
I agree this is a better approach. For the time being, I'm having some trouble with some edge cases around the screen marker plugin specifically, so I'm removing these changes from this PR while I adjust it so the resizing logic fix can be considered on its own.